docs/jsdocs.html

```diff
@@ -7259,15 +7259,15 @@ <h4>January 9th, 2022</h4>
 							<div class="spoiler">
 								<div class="settingsDiv">
 									<p><span class="hl">Prototype</span></p>
-									<p><em>MakeItem( socket, player, itemNum );</em></p>
+									<p><em>MakeItem( socket, player, itemNum, hueNum);</em></p>
```

Comment:
This isn't actually required (see https://github.com/UOX3DevTeam/UOX3/blob/320bfbe33b2b33b3f7e5621589f735f95dc78ec4/source/SEFunctions.cpp#L474 ) - it's an optional property that defaults to 0.

So perhaps `[, hueNum = 0]` to indicate optionality would be a good option.